app/views/talks_agent/summary.json.erb

```diff
@@ -0,0 +1,22 @@
+<%= {
```

Comment:
Is there a reason why this is in ERB and in a separate file?

Reply:
Separate file to separate of concerns, but doesn't have to be erb, though I have been using erb to render things like action names dynamically.